lib/model.js

```diff
@@ -401,7 +401,7 @@ class Model extends Entity {
         const args = Array.prototype.slice.apply(arguments);
 
         cb = args.pop();
-        id = parseId(id);
+        id = multiple ? id.map(parseId) : parseId(id);
         ancestors = args.length > 1 ? args[1] : undefined;
```

Comment:
test failed, because here multiple is not declared.
Can you add: `const multiple = is.array(id);` and squash into 1 commit?